src/main/java/racingcar/domain/Times.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.domain;
+
+public class Times {
+
+    int times;
+
+    public Times(String timesString) {
+        int times = validateIsPositive(Integer.parseInt(timesString));
+        this.times = times;
+    }
+
+    private int validateIsPositive(int times) {
+        if (times <= 0) {
+            throw new IllegalArgumentException("[ERROR] - 0보다 큰 정수를 입력해주세요.");
+        }
+        return times;
+    }
```

Comment:
아래와 같이 변경해보면 어떨까요? 🤔 
```suggestion
    private int times;

    public Times(int times) {
        validateTimes(times);
        this.times = times;
    }

    private void validateTimes(int times) {
        if (times <= 0) {
            throw new IllegalArgumentException("[ERROR] - 0보다 큰 정수를 입력해주세요.");
        }
    }
```